kebab/src/websocket/wrapper.ts

```diff
@@ -166,6 +166,10 @@ export const wrap = (connection: Connection) => ({
       connection.sendCall("room:deafen", { deafened: isDeafened }),
     userCreateBot: (username: string): Promise<CreateBotResponse> =>
       connection.sendCall(`user:create_bot`, { username }),
+    userSetContributions: (username: string, value: number) =>
+      connection.sendCall(`user:set_contributions`, { username, value }),
+    userSetStaff: (username: string, value: boolean) =>
+      connection.sendCall(`user:set_staff`, { username, value }),
```

Comment:
instead of 2 endpoints, create a single one called `user:admin_update` that can be used for more than 1 field

Reply:
done